docs/getting-started/github-apps/note--addl-commit.rst

```diff
@@ -0,0 +1,2 @@
+.. note::
+	Once the GitHub App is installed and a .dockstore.yml is present, please make sure to push one *additional* commit to your repository *after* the commit that adds the .dockstore.yml file. This helps make sure your workflows, tools, and services show up in Dockstore.
```

Comment:
FWIW, I think this is a little unclear. We need a commit of some kind after adding the repository in the github apps dialogue on github. That can either be "real" code or just adding/modifying the .dockstore.yml but there's nothing special about the .dockstore.yml that would disallow it from being a commit that would trigger the update process.

Reply:
In my experience the addition of .dockstore.yml triggers a successful install, but it is not until a successful push that it appears on Dockstore (see comment below).

Reply:
I interpreted this existing FAQ entry...
> If access was given to the wrong organization or repository, or this is your first time installing the app, you’ll need to push another commit after adding the correct repository to activate the sync to Dockstore.

...to mean that an additional commit is needed after any fresh install. But to clarify, you're saying that the fresh-install-needs-extra-commit thing applies iff a .dockstore.yml was present prior to the installation?

Reply:
@denis-yuen Seeking clarification before I close/edit this PR ^